pubsub/rocketmq/rocketmq.go

```diff
@@ -109,6 +112,9 @@ func (r *rocketMQ) setUpProducer() (mq.Producer, error) {
 	if r.metadata.GroupName != "" {
 		opts = append(opts, mqp.WithGroupName(r.metadata.GroupName))
```

Comment:
Should probably show a warning here that groupName is deprecated

Reply:
Done